src/api/webcontents/webcontents.ts

```diff
@@ -41,4 +41,13 @@ export class WebContents<T extends WebContentsEventMapping> extends EmitterBase<
             ignoreCache
         }, this.identity)).then(() => undefined);
     }
-}
\ No newline at end of file
+
+    public findInPage(searchTerm: string, options?: any): Promise<number> {
```

Comment:
The type of 'options' is FindInPageOptions ?

Reply:
added the type to the func